server/public/model/utils_test.go

```diff
@@ -948,6 +948,48 @@ func TestSanitizeUnicode(t *testing.T) {
 	}
 }
 
+func TestIsValidURI(t *testing.T) {
+	cases := []struct {
+		Description string
+		Input       string
+		Expected    bool
```

Comment:
1/5 that `Valid` is more intention revealing. 
```suggestion
		Valid    bool
```